build.sbt

```diff
@@ -121,8 +122,25 @@ def formattingPreferences = {
     .setPreference(SpacesAroundMultiImports, true)
 }
 
+val defaultMultiJvmOptions: List[String] = {
+  import scala.collection.JavaConverters._
+  // multinode.D= and multinode.X= makes it possible to pass arbitrary
+  // -D or -X arguments to the forked jvm, e.g. -Dmultinode.Xmx512m
```

Comment:
an example with `-D` would be `-Dmultinode.Djava.net.preferIPv4Stack=true` that would pass `-Djava.net.preferIPv4Stack=true` to the forked jvm

I'll add that example also.